src/app/fb/db.cljs

```diff
@@ -1,11 +1,40 @@
 (ns app.fb.db
-  (:require ["firebase/app" :refer [database]]
+  (:require [re-frame.core :refer [reg-fx dispatch]]
+            [clojure.walk :refer [keywordize-keys]]
+            ["firebase/app" :as firebase]
             [clojure.string :as str]))
 
-(defn db-ref
-  [path]
-  (.ref (database) (str/join "/" path)))
+(defn materialize-col-snap
```

Comment:
not sure about the `materialize` prefix, but i get the idea
maybe `colsnap->vec` ? (similarly `docsnap->map` ?)

Reply:
`colsnap->vec-of-maps` maybe?

Reply:
or `colsnap->maps`

Reply:
the reasoning is this is a conversion function, from one type/representation to the other and it is a good practice to name such functions with `->` inside
the hard part is naming the `to` here tho :)